src/components/hooks/use-breakpoints.tsx

```diff
@@ -0,0 +1,28 @@
+import useMediaQuery from './use-media-query'
+import { breakpoints } from 'themes/theme.breakpoints'
+
+export type TUseBreakpoints = {
```

Comment:
@michio-rework I don’t think we need to export it, If we need it somewhere we can use `ReturnType<typeof useBreakpoints>`